sample_ocdsdata_config.ini

```diff
@@ -0,0 +1,7 @@
+; sample_ocdsdata_config.ini
+[DBHOST]
+HOSTNAME = localhost
+PORT = 5432
+USERNAME = ocdsdata
+PASSWORD = ocdsdata
```

Comment:
Can we change this to changeme? I can imagine being lazy and new to Postgres and just setting this as the password!